plugins/module_utils/scale.py

```diff
@@ -189,15 +194,12 @@ def _create_stream(self, resource, namespace, wait_time):
         """ Create a stream of events for the object """
         w = None
         stream = None
-        try:
-            w = watch.Watch()
-            w._api_client = self.client.client
-            if namespace:
-                stream = w.stream(resource.get, serialize=False, namespace=namespace, timeout_seconds=wait_time)
-            else:
-                stream = w.stream(resource.get, serialize=False, namespace=namespace, timeout_seconds=wait_time)
-        except KubernetesException:
-            pass
+        w = watch.Watch()
+        w._api_client = self.client.client
+        if namespace:
```

Comment:
It looks like these two branches are identical, we can probably just remove this check.

Reply:
I'm moving this to a separate PR for scale testing